docs/install-agent.md

```diff
@@ -171,6 +199,25 @@ If you missed it in the "Provision a Server" page, we recommend you use Ubuntu 2
 
     Once you've run our test build, you need to run the steps for systemd mentioned above.
 
+### ZFS configuration
```

Comment:
This page also seems a bit orphaned.. perhaps you can bring your table down here and instead make it more about VM snapshot storage options?